exchanges/exchange.go

```diff
@@ -881,6 +881,9 @@ func (e *Base) SetAPIURL(ec *config.ExchangeConfig) error {
 	if ec.APIURL != config.APIURLNonDefaultMessage {
 		e.APIUrl = ec.APIURL
 	}
+	if !strings.Contains(e.APIUrl, "https") {
+		log.Warnf("Connection could be compromised, use at your own risk\n")
```

Comment:
Please mention the exchange name here and a less ambiguous message like:

"%s is using HTTP instead of HTTPS for API functionality, an attacker could eavesdrop this connection. Use at your own risk"